src/Symfony/Component/Security/Http/Firewall/UsernamePasswordJsonAuthenticationListener.php

```diff
@@ -73,6 +73,11 @@ public function __construct(TokenStorageInterface $tokenStorage, AuthenticationM
     public function handle(GetResponseEvent $event)
     {
         $request = $event->getRequest();
+        if (false !== strpos($request->getRequestFormat(), 'json')
+            || false !== strpos($request->getContentType(), 'json')
+        ) {
+            return;
+        }
```

Comment:
I think this should be:

```php
if (false === strpos($request->getRequestFormat(), 'json') && false === strpos($request->getContentType(), 'json')) {
    return;
}
```

Skipping only if both request format and content type do not contain `json`. It makes your tests green.
If we want to skip if only one of them does not contain `json`, you have to set the `_format` in all the existing tests in addition to the content type.